h/static/scripts/directive/annotation.coffee

```diff
@@ -281,7 +281,7 @@ AnnotationController = [
     this.render = ->
       # Extend the view model with a copy of the domain model.
       # Note that copy is used so that deep properties aren't shared.
-      angular.extend @annotation, angular.copy model
+      @annotation = angular.extend {}, angular.copy model
```

Comment:
Yes, indeed. Hadn't read the docs for `angular.copy()` - it creates the destination object implicitly.